meta-lmp-base/recipes-core/initrdscripts/lmp-recovery-framework.bb

```diff
@@ -0,0 +1,36 @@
+SUMMARY = "Linux microPlatform recovery modules"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://COPYING.MIT;md5=838c366f69b72c5df05c96dff79b35f2"
+
+inherit allarch
+
+SRC_URI = " \
+	file://uboot_env \
+	file://udhcpc \
+	file://image_download"
+
+S = "${WORKDIR}"
+
+do_install() {
+	install -d ${D}/recovery.d
+	install -m 0755 ${WORKDIR}/uboot_env.sh ${D}/recovery.d/10-uboot_env
```

Comment:
The `uboot_env.sh` file  is previews installed conditioned to `ubootenv` in `SOTA_CLIENT_FEATURES` so is better to keep it installed in the same way.
```
if ${@bb.utils.contains('SOTA_CLIENT_FEATURES', 'ubootenv', 'true', 'false', d)}; then
    install -m 0755 ${WORKDIR}/uboot_env.sh ${D}/recovery.d/10-uboot_env
fi
```